frontend/src/pages/modelRegistry/screens/utils.ts

```diff
@@ -38,15 +41,15 @@ export const mergeUpdatedLabels = (
   return customPropertiesCopy;
 };
 
-// Retrives the customProperties that are not labels (they have a defined string_value).
+// Retrives the customProperties that are not special (_RegisteredFrom) or labels (they have a defined string_value).
```

Comment:
small nit: Retrives -> Retrieves Not sure why I didn't catch that earlier. can be ignored coz this is a comment

Reply:
i'll sneak that into my next pr ;p

Reply:
updated along with the name change stuff mike wanted